Source/Core/Common/BitUtils.h

```diff
@@ -99,4 +99,26 @@ constexpr Result ExtractBits(const T src) noexcept
 
   return ExtractBits<T, Result>(src, begin, end);
 }
+
+///
+/// Verifies whether the supplied value is a valid bit mask of the form 0b00...0011...11.
+/// Both edge cases of all zeros and all ones are considered valid masks, too.
+///
+/// @param  mask The mask value to test for validity.
```

Comment:
Me neither, I was just copying the same style that was already used in the file.
If you point me to a file that demonstrates the recommended style, I can change the whole file, if desired.